packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -82,19 +92,52 @@ class _AndroidViewState extends State<AndroidView> {
     );
   }
 
-  @override
-  void initState() {
-    super.initState();
+  void _initializeOnce() {
+    if (_initialized) {
+      return;
+    }
+    _initialized = true;
+    assert(widget.layoutDirection != null || debugCheckHasDirectionality(context));
```

Comment:
Can't you just _layoutDirection = _findLayoutDirection() here?

Reply:
done